admin_guide/high_availability.adoc

```diff
@@ -14,34 +14,37 @@ toc::[]
 This topic describes how to set up highly-available services on your
 {product-title} cluster.
 
-The Kubernetes
-xref:../architecture/core_concepts/deployments.adoc#replication-controllers[replication
-controller] ensures that the deployment requirements, in particular the number
-of replicas, are satisfied when the appropriate resources are available. When
-run with two or more replicas, the
-xref:../architecture/core_concepts/routes.adoc#routers[router] can be resilient
-to failures, providing a highly-available service. Depending on how the router
-instances are discovered (via a service, DNS entry, or IP addresses), this could
-impose operational requirements to handle failure cases when one or more router
-instances are "unreachable".
-
-For some IP-based traffic services, virtual IP addresses (VIPs) should always be
-serviced for as long as a single instance is available. This simplifies the
-operational overhead and handles failure cases gracefully.
-
-IMPORTANT: Even though
-a service is highly available, performance can still be affected.
+IP Failover (IPF) provides high availibilty for a service in the cluster.
```

Comment:
This whole section is wrong.  The original was explaining the Kubernetes service proxy that seamlessly load-balances and provides IPF throughout the cluster.